maquetado/scss/detail.scss

```diff
@@ -0,0 +1,64 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+%equal-font-heights {
+ 
+  font-family: Roboto, sans-serif;;
+  display: flex;
+  align-items:baseline;
+
+}
+
+$color-1: #828282;
+$color-2: #000;
+$color-3: #BED23C;
+$color-4: #FFF;
+
+.book {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+  height: 416px;
+  width: 900px;
+  background-color: $color-4;
+  box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
+}
+
+.book-cover {
+  margin: 24px 52px 24px 24px;
+  height: 368px;
+  width: 261px;
+}
+
+.badge {
+  height: 118.57px ;
```

Comment:
The `.57` barely makes a difference. Use `118` or `120`. Also, delete that space before the semicolon.